src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -84,7 +84,7 @@ namespace nupic {
                         Real minPctActiveDutyCycles=0.001,
                         UInt dutyCyclePeriod=1000,
                         Real maxBoost=10.0,
-                        Int seed=1,
+                        Int seed=-1,
```

Comment:
Yes, in Py there was `seed=-1`, in C++ `seed=1`, please decide which to go with. @scottpurdy @oxtopus